content/10.zk-stack/10.components/110.transaction-filtering.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Transaction filtering
+description: Learn how to filter transactions on the ZK chain, including L1→L2 and L2 transactions.
+---
+
+The ZK Chain operators can filter both the transactions initiated from L1 and the ones submitted via API directly to L2.
+
+### L1->L2 Transactions
```

Comment:
Sorry I missed these in the initial review, can you update all the H3 headers to H2?